master/internal/command/events.go

```diff
@@ -143,7 +143,7 @@ func canAccessCommandEvents(ctx *actor.Context, c echo.Context) error {
 	var ok bool
 	if spec.TaskType == model.TaskTypeTensorboard {
 		ok, err = AuthZProvider.Get().CanGetTensorboard(
-			reqCtx, curUser, spec.WorkspaceID)
+			reqCtx, curUser, []model.AccessScopeID{spec.WorkspaceID})
```

Comment:
question: wouldn't this need to know what experiments and trials the tensorboard is accessing?

Reply:
ah I hadn't seen https://github.com/determined-ai/determined/pull/5828/files?show-viewed-files=true&file-filters%5B%5D=#diff-279040d0f1b6037ff1f5ab76d57037ecb3d46b48467b8a2219edde3444d50ff5R143